homeassistant/components/vizio/media_player.py

```diff
@@ -159,6 +159,7 @@ def __init__(
         )
         self._device = device
         self._max_volume = float(device.get_max_volume())
+        self._attr_assumed_state = True
```

Comment:
Why is this changed?

Reply:
Yes, this gives us separate play/pause buttons rather than a single dynamic button (which won't actually work correctly because the Vizio API doesn't provide a playing/paused/idle status).